README.md

```diff
@@ -288,6 +288,15 @@ const headers = await fetch(url)
   .then(res => res.headers)
 ```
 
+##### Forbidden and Safelisted Header Names
+
+* https://fetch.spec.whatwg.org/#cors-safelisted-response-header-name
+* https://fetch.spec.whatwg.org/#forbidden-header-name
+* https://fetch.spec.whatwg.org/#forbidden-response-header-name
+* https://github.com/wintercg/fetch/issues/6
+
+The [Fetch Standard](https://fetch.spec.whatwg.org) requires implementations to exclude certain headers from requests and responses. Undici does not filter out these headers.
```

Comment:
Could you add a bit of an explanation on _why_ we do it?

Reply:
Not sure how to word it entirely...

"In browser environments, these headers are forbidden so the user agent remains in full control over them. In Undici, these constraints are removed to give more control to the user."? I took some of the wording directly from the spec without making it seem like this is insecure, but I'm not sure if it's enough.

Reply:
Also noticed that undici's filtering wasn't really spec compliant as it allowed `sec-` and `proxy-` headers anyways.